tiatoolbox/utils/misc.py

```diff
@@ -342,8 +353,9 @@ def mpp2objective_power(mpp):
 
 
 def contrast_enhancer(img, low_p=2, high_p=98):
-    """Enhancing contrast of the input image using intensity adjustment.
-       This method uses both image low and high percentiles.
+    """Enhances contrast of the input image using intensity adjustment.
```

Comment:
```suggestion
    """Enhance contrast of the input image using intensity adjustment.
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/7deae026e07b52b86558f194e9231b630c756b33